ci/playbooks/edpm_build_images/edpm_build_images_content_provider_run.yaml

```diff
@@ -17,13 +17,14 @@
         chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
         cmd: >-
           ansible-playbook ci/playbooks/edpm_build_images/edpm_build_images_content_provider.yaml
+          -i "{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_inventory.yml"
+          -e "@{{ ansible_user_dir }}/ci-framework-data/artifacts/zuul_vars.yml"
```

Comment:
the zuul vars were passed at the end in order to ensure they have precedence over everything else. Maybe good to reproduce that?